misc-tools/dj_make_chroot.in

```diff
@@ -336,7 +336,11 @@ mount --bind /dev/pts "$CHROOTDIR/dev/pts"
 # Prevent perl locale warnings in the chroot:
 export LC_ALL=C
 
-chroot "$CHROOTDIR" /bin/sh -c debconf-set-selections <<EOF
+in_chroot() {
+    chroot "$CHROOTDIR" "$@"
```

Comment:
I think it makes sense to also include `/bin/sh -c` here as all calls of `in_chroot` use that.